src/mame/drivers/vsmjtria.cpp

```diff
@@ -52,12 +52,14 @@ class vsmjtria_state : public driver_device
 {
 public:
 	vsmjtria_state(const machine_config &mconfig, device_type type, const char *tag) :
-		driver_device(mconfig, type, tag),
-		m_cpu(*this, "cpu%u", 0U),
-		m_gfxdecode(*this, "gfxdecode%u", 0U),
-		m_colorram(*this, "colorram%u", 0U),
-		m_videoram(*this, "videoram%u", 0U),
-		m_key{ { *this, "P1_KEY%u", 0U }, { *this, "P2_KEY%u", 0U } }
+		driver_device(mconfig, type, tag)
+		, m_cpu(*this, "cpu%u", 0U)
+		, m_gfxdecode(*this, "gfxdecode%u", 0U)
+		, m_colorram(*this, "colorram%u", 0U)
+		, m_videoram(*this, "videoram%u", 0U)
+		, m_key{ { *this, "P1_KEY%u", 0U }, { *this, "P2_KEY%u", 0U } }
+		, m_nvram(*this, "nvram%u", 0U)
+		, m_sw2(*this, "SW2")
```

Comment:
If you’re going to change this to prefix-style, be consistent about it and move the colon to the beginning of the followin line as well.